src/Provider/Provider.Web/Orchestrators/Part1/EmployerOrchestrator.cs

```diff
@@ -15,9 +18,15 @@ public EmployerOrchestrator(IProviderVacancyClient providerVacancyClient)
             _providerVacancyClient = providerVacancyClient;
         }
 
-        public async Task<EmployersViewModel> GetEmployersViewModelAsync(VacancyRouteModel vacancyRouteModel)
+        public async Task<EmployersViewModel> GetEmployersViewModelAsync(VacancyRouteModel vrm)
         {
-            var editVacancyInfo = await _providerVacancyClient.GetProviderEditVacancyInfoAsync(vacancyRouteModel.Ukprn);
+            var editVacancyInfo = await _providerVacancyClient.GetProviderEditVacancyInfoAsync(vrm.Ukprn);
+
+            if (editVacancyInfo.Employers.Any() == false)
```

Comment:
Cant we use !editVacancyInfo.Employers.Any() instead of ==false?

Reply:
People have said `== false` is easier is to read and use either.